components/card/card.scss

```diff
@@ -0,0 +1,33 @@
+.Card {
+    border: 1px solid var(--center-channel-color-08);
+    box-shadow: 0px 8px 24px rgba(0, 0, 0, 0.12);
+    border-radius: 4px;
+    background-color: var(--center-channel-bg);
+
+    & + .Card {
+        margin-top: 16px;
+    }
+}
+
+.Card__header {
+    margin: 12px 12px 0 12px;
+    padding-bottom: 12px;
+    
+    &.expanded {
```

Comment:
See comment above about making the `.expanded` state part of the accordion rather than the card.